src/components/FilledButton/FilledButton.stories.tsx

```diff
@@ -3,7 +3,22 @@ import { storiesOf } from '@storybook/react'
 import FilledButton from './FilledButton'
 import { action } from '@storybook/addon-actions'
 import { withKnobs } from '@storybook/addon-knobs'
+import HeaderButton from './HeaderButton'
+import DropdownHeaderButton from './DropdownHeaderButton'
+
 storiesOf('buttons/FilledButton', module)
```

Comment:
Connect a wallet is jumping when scaling. We dont have designs for mobile so for now just fix this jumping button.